src/Mobile/MauiProgram.cs

```diff
@@ -25,6 +24,17 @@ public static MauiApp CreateMauiApp()
 
         Barrel.ApplicationId = "dotnetpodcasts";
 
+        builder.Services.AddMauiBlazorWebView();
```

Comment:
Ok, so this line is appropriate for maui/perf branch that tracks dotnet/maui main...